commit c1761755915ab59f418b801a6901230f67877047
Author: Wolfgang Bauer <wbauer@tmo.at>
Date:   Wed Jan 8 09:29:53 2014 +0100

    Screenlocker: don't set the mouse cursor when grabbing the mouse
    
    Setting the cursor to ArrowCursor when calling XGrabPointer() prevents
    the Screen savers from blanking the mouse cursor.
    With this change they can blank the mouse cursor again.
    
    BUG: 311571
    BUG: 316459
    FIXED-IN: 4.11.6
    REVIEW: 114841

diff --git a/ksmserver/screenlocker/ksldapp.cpp b/ksmserver/screenlocker/ksldapp.cpp
index 9e94b06..3dfcc9e 100644
--- a/ksmserver/screenlocker/ksldapp.cpp
+++ b/ksmserver/screenlocker/ksldapp.cpp
@@ -239,7 +239,7 @@ bool KSldApp::grabMouse()
                    EnterWindowMask | LeaveWindowMask
     int rv = XGrabPointer( QX11Info::display(), QApplication::desktop()->winId(),
             True, GRABEVENTS, GrabModeAsync, GrabModeAsync, None,
-            QCursor(Qt::ArrowCursor).handle(), CurrentTime );
+            None, CurrentTime );
 #undef GRABEVENTS
 
     return (rv == GrabSuccess);
